assets/js/components/Dashboard/NewPaMessage/StationsAndZones/SelectZonesPage.tsx

```diff
@@ -117,15 +117,17 @@ const SelectZonesPage = ({
     };
   }, [selectedRouteFilter]);
 
-  const filteredPlaces = useMemo(() => {
-    const placesWithSelectedSigns = places.filter((p) =>
+  const initialPlacesWithSelectedSigns = useMemo(() => {
```

Comment:
Just a note, `useMemo` is a good tool for fixing specific kinds of performance issues (measurably slow re-renders after frequent, unrelated state changes, e.g. keyboard events). However, it adds a bunch of extra complexity and statefulness in the form of the cache, so it's best avoided unless we know we need it.

Reply:
Thanks for the tip. I feel like I've been a bit too eager to use `useMemo` just because I don't need the setter from `useState`. Changed this and another variable into `useState` without a setter instead.

Reply:
If you don't ever need to set the state and don't want the setter cluttering things up, `const [value] = useState(...)` works. Though it really feels like it shouldn't if you're used to Elixir's destructuring.